react/features/base/avatar/functions.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import _ from 'lodash';
+
+const AVATAR_COLORS = [
```

Comment:
Can we have some more? 5 seeems to low.

Reply:
I agree. We can have as many as we want, but we're waiting for Tibi to provide final colors. Yana said that we can go with these for the time being. There will be a folow up "color clean up" PR soon that will change other colors as well